config/postgres/sample_templated_config.xml

```diff
@@ -19,6 +19,24 @@
 
     <query_templates_file>data/templated/example.xml</query_templates_file>
 
+    <!-- The anonymization configuration -->
+    <anonymization>
+        <table name="item" epsilon="1.0" pre_epsilon="0.0" algorithm="aim">
+            <!-- Column categorization -->
+            <droppable>i_id</droppable>
+            <categorical>i_name,i_data,i_im_id</categorical>
+            <continuous>i_price</continuous>
+            <!-- Continuous column fine-tuning -->
+            <continuousConfig>
```

Comment:
```suggestion
            <continuous_config>
```

nit: we have a mix of camel and underscore in here atm I know, but I think it's just one case which we can separately fix, so let's not make that worse here.